src/tests/radmin/all.mk

```diff
@@ -40,8 +40,8 @@ $(OUTPUT)/%: $(DIR)/% test.radmin.radiusd_kill test.radmin.radiusd_start
 		echo "--------------------------------------------------"; \
 		echo "$(RADIUSD_RUN)"; \
 		echo "$(RADMIN_BIN) -q -f $(RADMIN_SOCKET_FILE) > $(FOUND) < $<"; \
-		$(MAKE) $(TEST).radiusd_kill; \
-		exit 1;\
+		$(MAKE) test.radmin.radiusd_kill; \
```

Comment:
why is this preferred?  I'm happy to leave it as $(TEST)

Reply:
I would like too. but, does not expand.

```
[jpereira@sugarloaf freeradius-server.git]$ make Q= test.radmin | grep radiusd_kill
if [ -f build/tests/radmin/radiusd.pid ]; then ret=0; if ! ps `cat build/tests/radmin/radiusd.pid` >/dev/null 2>&1; then rm -f control-socket; echo "FreeRADIUS terminated during test called by test.radmin.radiusd_kill"; echo "GDB output was:"; cat "build/tests/radmin/gdb.log"; echo "--------------------------------------------------"; tail -n 40 "build/tests/radmin/gdb.log"; echo "Last entries in server log (build/tests/radmin/gdb.log):"; ret=1; fi; if ! kill -TERM `cat build/tests/radmin/radiusd.pid` >/dev/null 2>&1; then ret=1; fi; exit ; fi
		/usr/local/bin/gmake .radiusd_kill; \
		/usr/local/bin/gmake .radiusd_kill; \
		/usr/local/bin/gmake .radiusd_kill; \
		/usr/local/bin/gmake .radiusd_kill; \
		/usr/local/bin/gmake .radiusd_kill; \
		/usr/local/bin/gmake .radiusd_kill; \
		/usr/local/bin/gmake .radiusd_kill; \
		/usr/local/bin/gmake .radiusd_kill; \
```

Reply:
Even I've tried to use double reference but without success. e.g: `$$(TEST).radiusd_kill`

Reply:
Ah OK.  More boilermake magic.